amy/fiscal/views.py

```diff
@@ -509,7 +509,7 @@ def get_initial(self) -> Dict[str, Any]:
             "consortium": self.membership.consortium,
             "public_status": self.membership.public_status,
             "variant": self.membership.variant,
-            "agreement_start": self.membership.agreement_end,
+            "agreement_start": self.membership.agreement_end + timedelta(days=1),
```

Comment:
Wouldn't it be better if the agreement ended one day before full year from the start?

For example, agreement starts on April 1st 2022, and ends on March 31th, 2023. The new agreement will start also on April 1st 2023.

Reply:
@pbanaszkiewicz to confirm the end date for membership